dkist/utils/inventory.py

```diff
@@ -43,6 +43,7 @@ def __missing__(self, key):
         "originalFrameCount": "Level 0 Frame count",
         "primaryExperimentId": "Primary Experiment ID",
         "primaryProposalId": "Primary Proposal ID",
+        "productId": "Product ID",
```

Comment:
Looking at the inventory fields, it looks like there's also `spectralLines` and `isManuallyProcessed` which we should add to this mapping